<launch>

  <arg name="image"/>

  <node name="$(anon marker_detector)" pkg="pattern_pose_estimation" type="marker_detector_node" output="screen">
    <remap from="image" to="$(arg image)" />
    <rosparam command="load" file="$(find pattern_pose_estimation)/cfg/spiral_marker.yaml" />
    <param name="marker_frame" value="marker"/>
	<param name="use_history" type="bool" value="true"/>
	<param name="cache_camera_info" type="bool" value="true"/>
	<param name="lazy" type="bool" value="false"/>
	<param name="rectified" type="bool" value="true"/>
	<param name="show_debug_image" value="true"/>
	<param name="publish_tf" value="false"/>
  </node>

  <node name="marker_filter" pkg="pattern_pose_estimation" type="marker_filter_node" output="screen">
	<param name="publish_tf" value="true"/>
	<param name="filter_mode" value="mbags"/>
  </node>

</launch>

